components/user_settings/sidebar/user_settings_sidebar.jsx

```diff
@@ -185,39 +255,108 @@ export default class UserSettingsSidebar extends React.Component {
                     updateSection={this.updateSection}
                 />
             );
+        } else {
+            contents = (
+                <SettingItemMin
+                    title={
+                        <FormattedMessage
+                            id='user.settings.sidebar.autoCloseDMTitle'
+                            defaultMessage='Automatically Close Direct Messages'
+                        />
+                    }
+                    describe={this.renderAutoCloseDMLabel(this.state.settings.close_unused_direct_messages)}
+                    section={'autoCloseDM'}
+                    updateSection={this.updateSection}
+                />
+            );
         }
 
         return (
-            <SettingItemMin
-                title={
-                    <FormattedMessage
-                        id='user.settings.sidebar.autoCloseDMTitle'
-                        defaultMessage='Automatically Close Direct Messages'
-                    />
-                }
-                describe={this.renderAutoCloseDMLabel(this.state.settings.close_unused_direct_messages)}
-                section={'autoCloseDM'}
-                updateSection={this.updateSection}
-            />
+            <React.Fragment>
+                {contents}
+                <div className='divider-dark'/>
+            </React.Fragment>
         );
     };
 
-    renderUnreadLabel = (value) => {
-        if (value === 'true') {
-            return (
+    renderOrganizationLabel = () => {
+        const {
+            grouping,
+            unreadsAtTop,
+            favoriteAtTop,
+            sorting,
+        } = this.props.sidebarPreference;
+
+        const messages = [];
+
+        if (grouping === 'by_type') {
+            messages.push(
+                <FormattedMessage
+                    key='by_type'
+                    id='user.settings.sidebar.groupByTypeShort'
+                    defaultMessage='Group by channel type'
+                />
+            );
+        } else {
+            messages.push(
                 <FormattedMessage
-                    id='user.settings.sidebar.showUnreadSection'
-                    defaultMessage='At the top of the channel sidebar'
+                    key='none'
+                    id='user.settings.sidebar.groupByNoneShort'
+                    defaultMessage='No grouping'
                 />
             );
         }
 
-        return (
+        let sortingId;
+        let sortingDefaultMessage;
+        if (sorting === 'alpha') {
+            sortingId = 'user.settings.sidebar.sortAlphaShort';
```

Comment:
Can you wrap each of these IDs like `t('user.settings.sidebar.sortAlphaShort')` using `import {t} from 'utils/i18n'`?

That's needed so that the script that manages translation strings knows that these are IDs that should exist in `en.json`. Without that (or something like `Utils.localizeMessage` or `<FormattedMessage/>`), those strings will be removed from `en.json` on the next run.